.github/workflows/release.yml

```
@@ -0,0 +1,46 @@
+name: Release All SDKs
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  package-and-release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Determine version
+        id: meta
+        run: |
+          ref="${GITHUB_REF}"
+          sha="${GITHUB_SHA}"
+          version="${ref#refs/tags/}"
```

do you mean this here?

```suggestion
          version="${ref}#refs/tags/"
```